### PR TITLE
Add documenation how to restore Service Catalog Etcd Cluster

### PR DESCRIPTION
**Pull request by mszostok**
Confirm these statements before you submit your pull request:

- [x] I have read and submitted the required [Contributor Licence Agreements](https://github.com/kyma-project/community/blob/master/CONTRIBUTING.md#agreements-and-licenses).
- [x] This pull request follows the contributing guidelines, recommended Git workflow, and templates.
- [x] I have tested my changes.
- [x] I have updated the relevant documentation.
---

**NOTE:**  I cannot use braces in bash variable name because there is a bug in docs Jenkins job so I need to use in snippets this $var instead of this ${var}. I know that this not follows the best practices.

### CODE REVIEW

File: docs/service-catalog/docs/014-details-etcd-database.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+5. Export the **SECRET_NAME** environment variable with the Secret name to the ABS. Get the Secret name from [here](../../../resources/core/values.yaml#L34):
+```bash
+export SECRET_NAME={SECRET_NAME_FROM_VALUES_YAML}
+```
+
+6. Create the EtcdRestore Custom Resource which triggers a restore process:
+```bash
+sed -e "s|<full-abs-path>|$ABS_PATH|g" \
+    -e "s|<abs-secret>|$SECRET_NAME|g" \
+    assets/etcd-restore/restore-cr.yaml \
```
Comment (aszecowka) [inline docs/service-catalog/docs/014-details-etcd-database.md]: minor: maybe file name should indicate that it is a template?

File: docs/service-catalog/docs/014-details-etcd-database.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    | kubectl create -f -
+```
+
+Now the etcd-restore-operator restores a new cluster from the backup
+
+7. See the status of the Pods and wait until all of them are ready. Check the [number][sc-etcd-cluster-no] of the Pods which should be in `RUNNING` state before going to the next step:
```
Comment (aszecowka) [inline docs/service-catalog/docs/014-details-etcd-database.md]: Maybe, we can add information what is currently going on, for example "Now, restore operator is going to kill currently running etcd pods and create anew ones."

File: docs/service-catalog/docs/014-details-etcd-database.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+> **NOTE:** The [etcd-operator][etcd-operator] is Namespace-scoped and is installed only in `kyma-system` Namespace.
+
+## Details
+
+This section describes the backup and restore processes of the `etcd` cluster for the Service Catalog.
+
+### Backup
+
+To execute the backup process the following values need to be set in the [core][core-chart-values] chart:
```
Comment (klaudiagrz) [inline docs/service-catalog/docs/014-details-etcd-database.md]: `To execute the backup process, the following values must be set in the [core][core-chart-values] chart:`

File: docs/service-catalog/docs/014-details-etcd-database.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+
+| Property name              | Description |
+|---------------------------------------------------|---|
+| **etcd-operator.backupOperator.enabled**            | If set to true, the [etcd-operator][etcd-operator-chart] chart installs the [etcd-backup-operator][etcd-backup-operator-chart]. The etcd-operator also creates the [Secret][abs-creds] with the **storage-account** and **storage-key** keys.  |
+| **etcd-operator.backupOperator.abs.storageAccount** | The name of the storage account for the Azure Blob Storage (ABS). It stores the value for the **storage-account** Secret key. |
+| **etcd-operator.backupOperator.abs.storageKey**     | The key value of the storage account for the ABS. It stores the value for the **storage-key** Secret key. |
+| **global.etcdBackupABS.containerName**              | The ABS container to store the backup. If set, the Service Catalog [sub-chart][sc-backup-sub-chart] installs the CronJob which executes periodically the [Etcd Backup][etcd-backup-app] application. For more information on how to configure the backup CronJob, see the [Etcd Backup][etcd-backup-app-readme] documentation. |
```
Comment (klaudiagrz) [inline docs/service-catalog/docs/014-details-etcd-database.md]: The first link is broken :(

File: docs/service-catalog/docs/014-details-etcd-database.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+
+> **NOTE:** If you to set the **storageAccount**, **storageKey**, and **containerName** properties, the **etcd-operator.backupOperator.enabled** must be set to `true`.
```
Comment (klaudiagrz) [inline docs/service-catalog/docs/014-details-etcd-database.md]: > **NOTE:** If you ~to~ set / want to set

File: docs/service-catalog/docs/014-details-etcd-database.md (hunk 2 above)
Comment (klaudiagrz) [inline docs/service-catalog/docs/014-details-etcd-database.md]: ``in the `RUNNING` state``

File: docs/service-catalog/docs/014-details-etcd-database.md [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+```bash
+kubectl delete pod -n kyma-system -l app=core-catalog-apiserver
+```
+
+9. Restart the Service Catalog `controller-manager` Pod:
+```bash
+kubectl delete pod -n kyma-system -l app=core-catalog-controller-manager
+```
+
+10. Clean-up the etcd-restore-operator and EtcdRestore CR.
```
Comment (klaudiagrz) [inline docs/service-catalog/docs/014-details-etcd-database.md]: : at the end

File: docs/service-catalog/docs/014-details-etcd-database.md [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+```
+
+4. Export the **ABS_PATH** environment variable with the path to the last successful backup file.
+```bash
+export ABS_PATH=$(kubectl get cm -n kyma-system sc-recorded-etcd-backup-data -o=jsonpath='{.data.abs-backup-file-path-from-last-success}')
+```
+
+> **NOTE:** The ConfigMap name is defined [here][sc-backup-sub-chart] as the **APP_BACKUP_CONFIG_MAP_NAME_FOR_TRACING**.
+
+5. Export the **SECRET_NAME** environment variable with the Secret name to the ABS
```
Comment (klaudiagrz) [inline docs/service-catalog/docs/014-details-etcd-database.md]: Dot at the end

File: docs/service-catalog/docs/014-details-etcd-database.md [hunk 8]
```diff
@@ -0,0 +1,108 @@
...
+
+6. Create the EtcdRestore Custom Resource which triggers a restore process:
+```bash
+sed -e "s|<full-abs-path>|$ABS_PATH|g" \
+    -e "s|<abs-secret>|$SECRET_NAME|g" \
+    assets/etcd-restore/restore-cr.tpl.yaml \
+    | kubectl create -f -
+```
+
+Now the etcd-restore-operator restores a new cluster from the backup
```
Comment (klaudiagrz) [inline docs/service-catalog/docs/014-details-etcd-database.md]: Dot at the end

File: docs/service-catalog/docs/014-details-etcd-database.md [hunk 9]
```diff
@@ -0,0 +1,108 @@
...
+
+> **NOTE:** The [etcd-operator][etcd-operator] is Namespace-scoped and is installed only in `kyma-system` Namespace.
+
+## Details
+
+This section describes the backup and restore processes of the `etcd` cluster for the Service Catalog.
+
+### Backup
+
+To execute the backup process, the following values must be set in the [core][core-chart-values] chart:
```
Comment (klaudiagrz) [inline docs/service-catalog/docs/014-details-etcd-database.md]: ~the following values must be set~ -> `you must set the following values`

**Pull request merged**